manim/animation/transform.py

```diff
@@ -249,6 +297,25 @@ def __init__(
 
 
 class MoveToTarget(Transform):
+    """Examples
```

Comment:
Docstrings can't start with the example section, a one-line description is needed.

```suggestion
    """Transforms a mobject to the mobject stored in its ``target`` attribute.
    
    After calling the :meth:`~.Mobject.generate_target` method, the :attr:`target`
    attribute of the mobject is populated with a copy of it. After modifying the attribute,
    playing the :class:`.MoveToTarget` animation transforms the original mobject
    into the modified one stored in the :attr:`target` attribute.

    Examples
```